modules/search/class.jetpack-search-widget-filters.php

```diff
@@ -8,6 +8,7 @@ class Jetpack_Search_Widget_Filters extends WP_Widget {
 	protected $jetpack_search;
 
 	const DEFAULT_FILTER_COUNT = 5;
+	const DEFAULT_SORT = 'relevance_desc';
```

Comment:
Relevance ascending is never something we will be able to do, so no point in making the distinction. Relevance ascending basically means: score every document in the index and then start at the end. Doesn't really scale well. :)

Reply:
I had to include _desc here because of how I parse out the components of the "sort" field into "orderby" and "order" components. But you're right - we wouldn't offer that, which is why it's not one of the options :)

Reply:
Sounds good. :)